Shell/TheoryFinder.cpp

```diff
@@ -205,12 +205,9 @@ bool TheoryFinder::matchCode(const void* obj,
   unsigned cp = 0; // code pointer
 
   // the clause, if any
-  const Clause* clause;
+  const Clause* clause = nullptr;
   // the length of this clause
-  int clength;
-#ifdef VDEBUG
-  bool clength_assigned = false;
-#endif
+  int clength = 0;
```

Comment:
Again, can't convince the compiler that these are actually assigned, although I think they are. Just assign them, we'll get a quicker crash if `clause` is `nullptr` than if `clause` is unitialised.